immersion/kubeflow_pipelines/cicd/solutions/lab-03_vertex.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>After this is run, shall we add some text description of what the student should see as evidence/result of this successful build?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done!</p>